src/flinkStatementResultsManager.ts

```diff
@@ -207,6 +217,7 @@ export class FlinkStatementResultsManager {
 
       try {
         const currentResults = this._results();
```

Comment:
`current` is fuzzy concept in a function that _updates_ / _augments_ with additional results. Maybe respell these to `priorResults` and `priorRawResults`, so that then is more clear that we're FIFO:

`this._rawResults([...priorRawResults, ...(resultsData?.data ?? [])]);`